tensorflow/core/kernels/crop_and_resize_op.cc

```diff
@@ -425,7 +426,7 @@ class CropAndResizeGradImageOp : public AsyncOpKernel {
 namespace functor {
 template <typename T>
 struct CropAndResizeBackpropImage<CPUDevice, T> {
-  bool operator()(const CPUDevice& d,
+  bool operator()(const OpKernelContext* context,
```

Comment:
Could you please change the first parameter back to `CPUDevice`?
```suggestion
  bool operator()(const CPUDevice& d,
```

Reply:
Hmm...we can do that. But how to handle the following line:
https://github.com/tensorflow/tensorflow/blob/5173d9d9ff0cfcb5e175b801c21b10e2e3704dc3/tensorflow/core/kernels/crop_and_resize_op.cc#L530

Reply:
Oh, right. It has been a while I forgot what we need the context for. Then we'll need to fix the build errors instead.